felix/bpf/bpf.go

```diff
@@ -2349,3 +2350,67 @@ func IterPerCpuMapCmdOutput(output []byte, f func(k, v []byte)) error {
 	}
 	return nil
 }
+
+func ConfigureAndLoad(file string, data libbpf.GlobalData, mapsToBePinned ...string) (*libbpf.Obj, error) {
```

Comment:
```suggestion
func LoadFile/Object(file string, data libbpf.GlobalData, mapsToBePinned ...string) (*libbpf.Obj, error) {
```
My slight preference would be to avoid somethingAndSomethingElse. We are loading the object file with given parameters, it is a function after all ;-)